### DISCUSSION TITLE
File Format

### DISCUSSION THREAD

**Discussion by blackwellj** [chuot/rdio-scanner#147] [Q&A]
What file formats does the IOS app work with? currently my RTLSDR-airband files are in mp3 and they dont work. either with the conversion option checked or not

**Comment by chuot** [comment]
I noticed the same where some MP3 files generated by RTLSDR-Airband would not be converted by FFMPEG and would be stored as is by Rdio Scanner. If you look closely in your logs (journalctl) for Rdio Scanner, you should see something like this:

```
ffmpeg version 4.4.1 Copyright (c) 2000-2021 the FFmpeg developers
  built with gcc 11 (GCC)
  configuration: --prefix=/usr --bindir=/usr/bin --datadir=/usr/share/ffmpeg --docdir=/usr/share/doc/ffmpeg --incdir=/usr/include/ffmpeg --libdir=/usr/lib64 --mandir=/usr/share/man --arch=x8>
  libavutil      56. 70.100 / 56. 70.100
  libavcodec     58.134.100 / 58.134.100
  libavformat    58. 76.100 / 58. 76.100
  libavdevice    58. 13.100 / 58. 13.100
  libavfilter     7.110.100 /  7.110.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  9.100 /  5.  9.100
  libswresample   3.  9.100 /  3.  9.100
  libpostproc    55.  9.100 / 55.  9.100
[mp3 @ 0x5623e55a9600] Format mp3 detected only with low score of 1, misdetection possible!
[mp3 @ 0x5623e55a9600] Invalid frame size (768): Could not seek to 893.
pipe:: Invalid argument
```

This also explains why these MP3 files are not played on mobile devices by the native app. I would recommend opening an issue on the RTLSDR-Airband GitHub page.
